commands/reaction-roles/notify.js

```diff
@@ -0,0 +1,47 @@
+// Imports the discord.js-commando library which is required for commands to work. 
+const { Command } = require('discord.js-commando');
+
+const Embed = {
+	/* This is the main body of the embedded message. The \n character will
+	add new lines for extra space in the command. */
+	"description": "React to this message to get the Notify role. \nReact to this message to get the Notify role. \nWe will still Notify all of important updates, but this role will notify you on release of new DevLogs and additional resources!",
+	
+	// The color of the embed is in a decimal format.
+	"color": 3077393,
+};
+
+client.on("message", async message => {
+    // Checking if the message author is a bot.
+    if (message.author.bot) return false;
+
+    // Getting the role by ID.
+	const channel = message.guild.channels.cache.get("732162017955414057");
+    const role1 = message.guild.roles.cache.get(channel);
+
+    // Creating a filter.
+    const filter = (reaction, user) => user.id == message.author.id;
+
+    // Awaiting for the embed message to be sent.
+    const reactionMessage = await message.channel.get(channel).send(Embed);
```

Comment:
Does this send the embed to the channel every time a none bot message is sent?

Reply:
looks like it does. I'll fix it